charts/velero/values.yaml

```diff
@@ -187,6 +190,16 @@ snapshotsEnabled: true
 deployRestic: false
 
 restic:
+  # Annotations to add to the Restic daemonset's pod template. Optional.
+  #
+  # If using kube2iam or kiam, use the following annotation with your AWS_ACCOUNT_ID
+  # and RESTIC_ROLE_NAME filled in:
+  #  iam.amazonaws.com/role: arn:aws:iam::<AWS_ACCOUNT_ID>:role/<RESTIC_ROLE_NAME>
+  podAnnotations: {}
```

Comment:
What's the difference to this https://github.com/vmware-tanzu/helm-charts/blob/1e74a1b/charts/velero/values.yaml#L12-L17?

The purpose is to separate the role of restic and velero?